mcmc/SampleSummary.cpp

```diff
@@ -2615,7 +2541,61 @@ void SampleSummary::StudyDIC() {
   const double DIC_stat = Dhat + 2 * p_D;
   MACH3LOG_INFO("Effective number of parameters following DIC formalism is equal to: {:.2f}", p_D);
   MACH3LOG_INFO("DIC test statistic = {:.2f}", DIC_stat);
-  MACH3LOG_INFO("******************************");
+}
+
```

Comment:
Since you refer to "equation 3" later on in the code, please put the reference in the header before the function declaration, or here in the implementation before the function declaration

Reply:
They are in header file this way it should be readable in doxygen
https://github.com/mach3-software/MaCh3/blob/01451f03fb9871fc15bcfded02aaf2940a6ed59e/mcmc/SampleSummary.h#L153